OpenCorePkg.dsc

```diff
@@ -398,10 +398,8 @@
   #
   MdeModulePkg/Universal/Disk/RamDiskDxe/RamDiskDxe.inf
 
-  CryptoPkg/Library/IntrinsicLib/IntrinsicLib.inf
-
```

Comment:
I added this in the current network boot PR, but I don't think it was needed (see below comment about similar line in OpenDuetPkg).

Reply:
https://github.com/acidanthera/OpenCorePkg/pull/556

Reply:
Okay, based on feedback from #556, I believe there is indeed no point in including these lines for another package's library, as this is now - but it would have made sense to include them before, when they were our package's library.